package.json

```diff
@@ -59,29 +44,24 @@
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "^1.7.0",
     "lintspaces-cli": "^0.3.0",
-    "lolex": "^1.5.0",
     "mocha": "^2.1.0",
     "mockery": "^1.7.0",
     "nightwatch": "^0.9.8",
     "node-fetch": "^1.6.0",
-    "node-sass": "^3.8.0",
     "nodemon": "^1.9.2",
     "npm-prepublish": "^1.2.1",
     "pa11y-ci": "^0.3.0",
-    "postcss-loader": "^0.9.1",
     "proxyquire": "^1.7.10",
-    "sass-loader": "^3.2.0",
     "semver": "^5.3.0",
     "sinon": "^1.17.4",
     "sinon-chai": "^2.8.0",
     "supertest": "^2.0.1",
-    "webpack": "^1.13.1"
+    "@financial-times/n-webpack": "^2.0.0"
```

Comment:
for now needs to still be a prod dependency. Even though it's a dev tool, when n-ui is installed by an app it will only install its dependencies, not devDependencies